man/tmpfiles.d.xml

```diff
@@ -51,7 +50,7 @@ p     /fifo/to/create                          mode user group -           -
 L     /symlink/to/create                       -    -    -     -           symlink/target/path
 c     /dev/char-device-to-create               mode user group -           -
 b     /dev/block-device-to-create              mode user group -           -
-# p+, L+, c+, b+ create target unconditionally, w+ appends to the file
+# p+, L+, c+, b+ create target unconditionally, w+ appends to the file, f+ creates or truncates the file
```

Comment:
If I continue refactoring the deprecation as proposed in https://github.com/systemd/systemd/pull/13862#issuecomment-547054647, I think we'd probably want to modify this table to include the `+` option inline somehow rather than making this comment longer. For example, rather than removing the `F` line, it could become this instead:

`f+     /file/to/create-or-truncate              mode user group -           content`

Reply:
Yes. Please split the comment up.

Reply:
I updated the original commit with an example of how I would clean up the comment in the man page by replacing lines in the table with the `+` versions. I went ahead and did the `H -> h+` refactor as well. If these changes seem to be moving in the right direction, I'll continue as proposed in https://github.com/systemd/systemd/pull/13862#issuecomment-547054647